src/go/k8s/config/crd/bases/toolkit.fluxcd.io/README.txt

```diff
@@ -1,2 +1,16 @@
-kustomize build https://github.com/fluxcd/source-controller//config/crd -o source-controller.yaml
-kustomize build https://github.com/fluxcd/helm-controller//config/crd -o helm-controller.yaml
+
```

Comment:
nit: Can we rename this to `README.md` as it contains markdown?

Reply:
ah good point.